www/templates/project-list.html

```diff
@@ -6,6 +6,11 @@
 
 {% block content %}
 
+<div class="row">
+ <div class="col-lg-12">
+  <h2>{% block title %}Project list{% endblock %}</h2>
+ </div>
+</div>
```

Comment:
I prefer we don't add this block. The spacing, alignment, and sizing don't feel good in the surroundings.

Reply:
I mean no `<h2>` in `<body>`. It's fine in `<head><title>...`.